anime-list-master.xml

```diff
@@ -992,7 +992,7 @@
     <name>Cowboy Bebop: Tengoku no Tobira</name>
     <mapping-list>
       <mapping anidbseason="0" tvdbseason="0">;1-0;2-0;3-0;4-0;</mapping>
-      <mapping anidbseason="1" tvdbseason="0">;1-1;2-1;3-1;4-1;5-1;6-1;</mapping>
+      <mapping anidbseason="1" tvdbseason="0">;1-3;2-3;3-3;4-3;5-3;6-3;</mapping>
```

Comment:
Only map anidb episode 1, the rest should not be mapped.

Reply:
Just followed what was already there

Reply:
I imagine there was a reason for being that way before.
Considering that AniDB episodes 2-6 are just re-edits of episode 1, it does seem more reasonable to have them all mapped to TVDB's only entry than mapped to 0

Reply:
Hama doesn't work that way. If you have AniDB episode 2 and then tell Hama to prioritize TVDB titles, every part will have the same title and thumbnail.

Reply:
I assumed HAMA was not the only project to use this list and this style of mapping might've been useful for another.
Regardless, just checked the notes on commit 8412c59 (thanks for those existing in the first place!) and I'm more comfortable switching to 0 now since even back then the goal was a null-map anyways.